android/app/src/main/java/im/status/ethereum/MainActivity.java

```diff
@@ -231,7 +231,7 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
         }
         if (grantResults.length == 1 && grantResults[0] == PackageManager.PERMISSION_GRANTED) {
             // Permission has been granted. Start camera preview Activity.
-            com.github.alinz.reactnativewebviewbridge.WebViewBridgeManager.grantAccess(requestCode);
+            //com.github.alinz.reactnativewebviewbridge.WebViewBridgeManager.grantAccess(requestCode);
```

Comment:
could you add a comment in code, why it was commented ?

Reply:
I will simply remove the line, it's not used any more.